### ISSUE TITLE
socket timed out

### ISSUE THREAD

**Issue by jitchavan** [seishun/node-steam#440]
I am not able to connect account. After logged debug event get following log:

```
socket closed
connecting to 72.165.61.188:27017
socket timed out
socket closed
connecting to 146.66.152.12:27018
socket timed out
socket closed
connecting to 72.165.61.185:27018
socket timed out
socket closed
connecting to 72.165.61.174:27018
socket timed out
```

Source Code as follow:
```
var Steam = require('steam');
var steamClient = new Steam.SteamClient();
var steamUser = new Steam.SteamUser(steamClient);

steamClient.connect();
steamClient.on('connected', function () {
        steamUser.logOn({
            account_name: ‘xxxxxxx’,
            password: ‘xxxxx’
        });
});

steamClient.on('debug', function (response) {
        console.log(response);
});
    
steamClient.on('logOnResponse', function (response) {
        console.log(response);
});

steamClient.on('error', function () {
        console.log('[ERROR] '+error);
});
```

Below are my installed versions.
```
Node Version: v10.8.0
NPM Version: 6.4.1
Steam Version: 1.4.0
OS: Mac

```
All are the updated versions. 

Please help me to find out issue. Not able to understand what is missing.

**Comment by scholtzm** [comment]
See:

**Comment by jitchavan** [comment]
Thanks. above issue is resolved by updating server list from #434 as @seishun suggested.
